setup.py

```diff
@@ -50,8 +50,8 @@ def read(fname):
     # different packages for different variants (e.g. mxnet-mkl and mxnet-cu90).
     install_requires=['sagemaker-inference==1.0.0'],
     extras_require={
-        'test': ['tox', 'flake8', 'pytest', 'pytest-cov', 'pytest-xdist', 'mock', 'sagemaker',
-                 'docker-compose', 'mxnet==1.4.0']
+        'test': ['tox', 'flake8', 'pytest', 'pytest-cov', 'pytest-xdist', 'mock',
+                 'sagemaker==1.23.0', 'docker-compose', 'awslogs']
```

Comment:
https://github.com/aws/sagemaker-mxnet-serving-container/blob/master/test/unit/test_default_inference_handler.py#L19

We're still using mxnet :(

Reply:
good catch - fixed